package.json

```diff
@@ -18,7 +18,6 @@
     "babel-plugin-dynamic-import-webpack": "^1.0.1",
     "babel-plugin-file-loader": "^1.0.0",
     "babel-plugin-import": "^1.1.0",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
```

Comment:
Also update yarn.lock?

Reply:
`yarn install` did not change yarn.lock in this case. I guess that some other packages still depend on async-to-generator which is why it's not removed from yarn.lock?